src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    body,
+    comments,
+    user,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <UserInfo user={user} />
+
+        <p className="PostInfo__body">
+          {body}
+        </p>
+
+        {comments.length > 0 ? (
```

Comment:
let's use !!comments.length or Boolean(comments.length) instead of > 0, easeier way to handle it + you won't receive any 0 on the page, if there's no comments